File: Python/02. Working with Numbers/46. Counting vowel and consonant.py
word = input("Enter the word to count Vowel and Consonent :")
lword = word.lower()
vowels = "aeiou"

vowelCount = 0
consonantCount = 0

for i in lword:
    if (i.isalpha()):
        if (i in vowels):
            vowelCount += 1
        else:
            consonantCount += 1

print("The", word, "has", vowelCount, "Vowels and", consonantCount, "Consonents")

File: Java/02. Working with Numbers/46. Counting vowel and consonant.java
import java.util.Scanner;

class Main {
    public static void main(String [] args) {
		Scanner userInput = new Scanner(System.in);
        System.out.println("Enter the word to count Vowel and Consonent :");
        String word = userInput.next();
        word = word.toLowerCase();
        String vowels = "aeiou";
        
        int vowelCount = 0;
        int consonantCount = 0;
        
        for (int i = 0; i < word.length(); i++) {
            char letter = word.charAt(i);

            if (vowels.indexOf(letter) != -1) {
                vowelCount++;
            } else {
                consonantCount++;
            }
        }
        
        System.out.println("Vowels: " + vowelCount);
        System.out.println("Consonants: " + consonantCount);
        userInput.close();
    }
}